Documentation/index.md

```diff
@@ -1,17 +1,46 @@
-Welcome to the MonoGame game library documentation hub.
+# Welcome to the MonoGame game library documentation hub.
 
 This area of the site contains the documentation on the API of MonoGame as well as how to use it to create great games.
-Note that this is a work in progress so there will be gaps in the documentation coverage. 
 
 If you cannot find what you need here you can also look at the [Microsoft XNA documentation](https://msdn.microsoft.com/en-us/library/bb200104.aspx). MonoGame is API compatible
-with [XNA](https://msdn.microsoft.com/en-us/library/bb203940.aspx) even down to the namespaces. So usually what works for XNA will work for MonoGame too. 
+with [XNA](https://msdn.microsoft.com/en-us/library/bb203940.aspx) even down to the namespaces. So usually what works for XNA will work for MonoGame too.
 
-Note that this documentation hub is built from the source code on every commit to the development branch. As such 
-it applies to the development builds available on the [Downloads](http://www.monogame.net/downloads) page. This may include new features which may
-not be available in the current stable release 
+The documentation site is made up of several major parts which are available as links at the top of the page.  
 
-### We Need Your Help!
+> If you are new to MonoGame, we highly recommend walking through the **[Getting Started](articles/getting_started/getting_started.md)** section to Install MonoGame and build your first game.
 
-Truly great open source projects require high quality documentation.  This is call for volunteers to help us make the MonoGame documentation truly great.  If you can help write tutorials, guides, code snippets, reference docs, video walkthroughs or just any improvement to our current documentation we could use your help!
+* [Introduction to MonoGame](articles/introduction/introduction.md)
+  
+  Provides helpful information and requirements for developing games with MonoGame.
+
+* [Getting Started guides](articles/getting_started/getting_started.md)
+
+  Provides introductory information about installing and getting started with developing games using MonoGame.
+
+* [Platform Specific help](articles/platform_specific/platform_specific.md)
```

Comment:
Let's change it to just `Platform Notes`.

Reply:
Also related to this....

![image](https://user-images.githubusercontent.com/103522/88600989-2d4bae00-d035-11ea-8d40-15dc09e839d1.png)

... be good for the case be consistent across all docs (using normal headline casing rules).  Also don't think a hyphen is need there.

Reply:
Sorted